ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-missing-evpn-vlan-bundle.yml

```diff
@@ -0,0 +1,48 @@
+# Enable vlan aware bundles
+evpn_vlan_aware_bundles: true
+
+type: l3leaf
+
+l3leaf:
+  defaults:
+    # LOOPBACK AND VTEP MANAGEMENT
+    loopback_ipv4_pool: 10.10.0.0/24
+    vtep_loopback_ipv4_pool: 10.11.0.0/24
+    vtep_loopback: Loopback1
+    # Possibility to prevent configuration of Tenant VRFs and SVIs. This allows support for centralized routing.
+    evpn_services_l2_only: true
+  nodes:
+    - name: failure-missing-evpn-vlan-bundle
+      id: 3
+      bgp_as: 65101
+      filter:
+        tags: [fabric, ipmi, storageProd]
+
+tenants:
+  - name: StorageProd
+    mac_vrf_vni_base: 10000
+    l2vlans:
+      - id: 10
+        name: storage_prod
+        tags: ['storageProd']
+      - id: 20
+        name: storage_prod
+        tags: ['storageProd']
```

Comment:
In favor of removing what is not needed in there in order to avoid maintaining things we don't need
```suggestion
```